### PR TITLE
FLUT-926192 - [Feature] Modified assist view name and updated API links

### PR DESCRIPTION
**Pull request by Hariram-SF4428**
### Feature description ###

- Task ID: [[FLUT-926192]](https://dev.azure.com/EssentialStudio/Mobile%20and%20Desktop/_workitems/edit/926192/), [[FLUT-926190]](https://dev.azure.com/EssentialStudio/Mobile%20and%20Desktop/_workitems/edit/926190/)
- Modified widget name in all pages, places and modified the GIF with width factor 0.95.
- Updated API link to all SfAIAssistView widget direct and class APIs, also added FT and SB link.

### Analysis and design ###

If there is an external design, link to its project documentation area.
If there is an internal discussion on the forum, provide the link.

### Solution description ###

Describe your code changes in detail for reviewers.

### Output screenshots ###

Post the output screenshots if an UI is affected or added due to this feature.

### Areas affected and ensured ###

List the areas are affected by your code changes.

### API Changes ###

List all API changes here (or just put None), example:

Added:
 - string ListView.GroupName { get; set; } //Bindable Property
 - int ListView.GroupId { get; set; } // Bindable Property
 - void ListView.Clear ();

Changed:
 - object ListView.SelectedItem => Cell ListView.SelectedItem

### Behavioral Changes ###

Describe any non-bug related behavioral changes that may change how users app behaves when upgrading to this version of the codebase.

### Test cases ###

Provide the unit testing written file details to understand the use cases considered in this implementation.
If there is no TDD (if it’s not possible to follow), provide the UI automation script location and the Excel file that contains the use cases considered in this implementation.
Provide the test cases Excel file alone if the feature cannot be automated in any case.

List out all the scenarios you have tested after include these changes. You can refer the predefined scenarios from below link.

https://syncfusion.atlassian.net/wiki/display/CHARTXAMARIN/Chart+Test+Plan

### Testbed sample location ###

Provide the test bed sample location where code reviewers can review the new feature’s behaviors. This depends on the CI process that your team follows. It can be from NPMCI, HockeyApp, staging site, local server, etc.

### Does it have any known issues?

If this feature contains any known issues, provide the proper details about the issues.

### Does it have memory leak?

Ensure the feature contains memory leak or not (if applicable).

### MR CheckList ###

- [ ] Does it follow the design [guidelines](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/)? It is mandatory that, we should not move the changes without reading this.
- [ ] Did UI automation passed without errors? If it is customer issue, make sure it is included in the IR automation.
- [ ] Properly working in Xamarin.Forms [previewer](https://docs.microsoft.com/en-us/xamarin/xamarin-forms/xaml/xaml-previewer?tabs=vswin).
- [ ] Ensured in iOS, Android, UWP and macOS (if supported).
- [ ] Have you added API comments if it is new API?
- [ ] Does it need localization? If so, update resx file in [GitHub](https://github.com/syncfusion/xamarin-localized-texts).
- [ ] Whether we have added new APIs? If so, whether it is as per the [guidelines](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/)?
- [ ] If changes are made in the Xamarin.Forms sample browser project file, whether it is included in the props file too? If not, your sample level changes will not reflect in the application published in the hockey app.
- [ ] Have you ensured the changes in Android API 19 and iOS 9?

### CODE REVIEW

File: Flutter/AIAssistView/AIAssistView-theme.md
Comment (VijayakumarMariappan) [inline Flutter/AIAssistView/AIAssistView-theme.md:1]: Why this file name is case sensitive?

File: Flutter/AIAssistView/images/overview/assist-overview.gif
Comment (VijayakumarMariappan) [inline Flutter/AIAssistView/images/overview/assist-overview.gif:1]: ai-assistview

Name always as this

File: Flutter/AIAssistView/AIAssistView-theme.md
Comment (VijayakumarMariappan) [reply Flutter/AIAssistView/AIAssistView-theme.md:1]: ai-assistview-theme.md

Comment (VijayakumarMariappan) [reply Flutter/AIAssistView/AIAssistView-theme.md:1]: Folder name also be small letter - check other widgets

ai-assistview = folder name

File: flutter-toc.html [hunk 1]
```diff
@@ -98,15 +98,15 @@
 	<li><a href="/Flutter/right-to-left">Right to Left (RTL)</a></li>
     <li><a href="/Flutter/themes">Themes</a></li>
     <li>
-		AssistView
+		AI AssistView
 		<ul>
-			<li><a href="/Flutter/assistview/overview">Overview</a></li>
-      <li><a href="/Flutter/assistview/getting-started">Getting Started</a></li>
```
Comment (VijayakumarMariappan) [inline flutter-toc.html:104]: In link - correct the widget name with casing

**Review by VijayakumarMariappan — CHANGES_REQUESTED** [review]

File: Flutter/AIAssistView/conversation-area.md [hunk 2]
```diff
@@ -238,7 +238,7 @@ The message author's avatar displays either an image or the initials of their na
         data: response,
         time: DateTime.now(),
         author: const AssistMessageAuthor(
-            id: '123-002',
```
Comment (VijayakumarMariappan) [inline Flutter/AIAssistView/conversation-area.md:241]: AI AssistView ID - Just mention 'ID'

**Review by VijayakumarMariappan — CHANGES_REQUESTED** [review]

File: Flutter/AIAssistView/AIAssistView-theme.md
Comment (Hariram-SF4428) [reply Flutter/AIAssistView/AIAssistView-theme.md:1]: Hi @VijayakumarMariappan sir,

I have modified the folder name AIAssistView to ai-assistview and renamed the file names AIAssistView-theme.md file to ai-assistview-theme, as per your suggestions. checked and ensured in all places of UG docs.

Regards,
Thankyou

File: flutter-toc.html (hunk 1 above)
Comment (Hariram-SF4428) [reply flutter-toc.html:104]: Hi @VijayakumarMariappan sir,

I have modified the folder name AIAssistView to ai-assistview in the toc file, checked and ensured in all places of UG docs.

Regards,
Thankyou

File: Flutter/AIAssistView/conversation-area.md (hunk 2 above)
Comment (Hariram-SF4428) [reply Flutter/AIAssistView/conversation-area.md:241]: Hi @VijayakumarMariappan Sir,

I have modified the AI AssistView ID to ID in the source conversation area page of UG docs as per your suggestions.

Regards,
Thankyou

File: Flutter/AIAssistView/images/overview/assist-overview.gif
Comment (Hariram-SF4428) [reply Flutter/AIAssistView/images/overview/assist-overview.gif:1]: Hi @VijayakumarMariappan Sir,

I have completed modifying the images folder name assistview-overview.gif to ai-assistview-overview.gif as per your suggestions.

Regards,
Thankyou

**Review by VijayakumarMariappan — APPROVED** [review]
Fine

**Pull request merged**